src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1042,6 +1049,9 @@ void cbmc_parse_optionst::help()
     " --yices                      use Yices\n"
     " --z3                         use Z3\n"
     " --refine                     use refinement procedure (experimental)\n"
+    " --incremental-smt2-solver cmd\n"
+    "                              Command to invoke external SMT solver for\n"
```

Comment:
"Command" -> c "command" for consistency

Reply:
done.